frontend/src/components/MainSearch/MainSearch.js

```diff
@@ -1,38 +1,50 @@
 import React, { Component } from 'react';
-import './MainSearch.css'
+import './MainSearch.css';
 
 class MainSearch extends Component {
-
   constructor(props) {
     super(props);
     this.state = {
-      searchTerm: ""
+      searchTerm: '',
     };
 
     this.handleChange = this.handleChange.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
   }
 
   handleChange(event) {
-    this.setState({searchTerm: event.target.value});
+    this.setState({ searchTerm: event.target.value });
   }
 
   handleSubmit(event) {
     event.preventDefault();
     this.props.setSearchTerm(this.state.searchTerm);
   }
 
-  render (){
+  render() {
     return (
       <div className="mainSearch container-fluid">
         <div className="row">
           <div className="searchSection">
-            <p className="findText">Find Recipes from Rescuing Leftover Cuisine</p>
-              <span className="search">
-                <form className="search-form" onSubmit={ this.handleSubmit }>
-                  <input type="text" className="search-input" placeholder="Search by keywords" onChange={ this.handleChange } />
-                </form>
-              </span>
+            <p className="findText">
+              Find Recipes from Rescuing Leftover Cuisine
+            </p>
+            <span className="search">
+              <form className="search-form" onSubmit={this.handleSubmit}>
+                <input
+                  type="text"
+                  className="search-input"
+                  placeholder="Search by keywords"
+                  onChange={this.handleChange}
+                />
+                <button type="submit" className="search-button">
```

Comment:
Can we add something to this so that the cursor changes to the clicky hand cursor on hover?